lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -67,16 +57,21 @@ def make_openshift_match_database
 
       ### API STUFF
       def each_source
+        source_types_by_id = {}
+        each_resource(sources_api_url_for("source_types")) { |source_type| source_types_by_id[source_type["id"]] = source_type }
+
         each_tenant do |tenant|
-          each_resource(url_for("source_types"), tenant) do |source_type|
-            collector_definition = collector_definitions[source_type["name"]]
-            next if collector_definition.nil?
-            each_resource(url_for("source_types/#{source_type["id"]}/sources"), tenant) do |source|
-              next unless endpoint = get_and_parse(url_for("sources/#{source["id"]}/endpoints"), tenant)["data"].first
-              next unless authentication = get_and_parse(url_for("authentications?resource_type=Endpoint&resource_id=#{endpoint["id"]}"), tenant)["data"].first
-              auth = authentication_with_password(authentication["id"], tenant)
-              yield source, endpoint, auth, collector_definition
-            end
+          each_resource(topology_api_url_for("sources"), tenant) do |source_stub|
```

Comment:
I think `topo_source` (or something) would be more clear to me here rather than `source_stub`.

Reply:
:+1: Done